### PR TITLE
feature RSTREAM 109 match prod code

### PR DESCRIPTION
**Pull request by czirker**

### CODE REVIEW

File: api/stats/index.js [hunk 1]
```diff
@@ -3,14 +3,79 @@
...
+let compressionThreshold = 100000; // 100k
 
 exports.handler = require("leo-sdk/wrappers/resource")(async (event, context, callback) => {
     await request.authorize(event, {
         lrn: 'lrn:leo:botmon:::',
         action: "stats",
         botmon: {}
     });
+
+    function findBigStrings(obj, prefix = "") {
```
Comment (czirker) [inline api/stats/index.js:17]: @jgrantr Do you know why this was added?

File: api/stats/index.js [hunk 2]
```diff
@@ -3,14 +3,79 @@
...
     stats(event, (err, data) => {
-        callback(err, (data || {}).stats)
+        let stats = (data || {}).stats;
+        if (stats) {
+            let responseBody = JSON.stringify(stats);
+            console.log(`response body length = ${responseBody.length}`);
+            if (responseBody.length > 6000000) {
+                findBigStrings(stats);
+            }
+            let isBase64Encoded = false;
```
Comment (czirker) [inline api/stats/index.js:48]: @jgrantr is this still needed if we have MinimumCompressionSize set?

File: api/stats/index.js (hunk 1 above)
Comment (jgrantr) [reply api/stats/index.js:17]: @czirker I think this was to track do-nothings that were too big.

File: api/stats/index.js (hunk 2 above)
Comment (jgrantr) [reply api/stats/index.js:48]: @czirker If we are compressing, probably not, unless we end up with something too big for even compressing, and then we may want to be able to track it down.

File: api/stats/index.js (hunk 1 above)
Comment (czirker) [reply api/stats/index.js:17]: We will leave it in for now

File: api/stats/index.js (hunk 2 above)
Comment (czirker) [reply api/stats/index.js:48]: We will leave it in for now

**Review by mecham-lynn — APPROVED** [review]

**Pull request merged**